PARCEL_2_RFC.md

```diff
@@ -0,0 +1,1285 @@
+# Parcel 2 RFC
+
+This RFC is written as documentation for the project, our documentation should
+be detailed enough to work as a spec, and it's easier to write it once this way
+than to approve a spec and start again on the documentation from scratch.
+
+Everything in Parcel should be documented here, if it is not documented it will
+not be part of Parcel 2.
+
+---
+
+## Introduction
+
+Parcel is a compiler for all your code, regardless of the language or toolchain.
+
+Parcel takes all of your files and dependencies, transforms them, and merges
+them together into a smaller set of output files that can be used to run your
+code.
+
+Parcel supports many different languages and file types out of the box, from
+web technologies like HTML, CSS, and JavaScript, to lower level languages like
+Rust, and anything that compiles to WebAssembly (WASM), to assets like images,
+fonts, videos, and more.
+
+Parcel makes your code portable, you can build your code for different
+environments, for the web for your server, or for an app. You can even build
+multiple targets at once and have them live update as you make changes.
+
+Parcel is fast and predictable. It compiles all of your files in isolation in
+parallel inside workers, caching all of them as it goes along. Caches are
+stable across machines and are only affected by the files and configs within
+your project (unless you want to pass specific environment variables).
+
+## Getting Started
+
+Before we get started, you'll need to install Node and Yarn (or npm) and create
```

Comment:
What is the minimum node version that is supported?

Reply:
It will always be whatever the lowest active LTS version of Node is.